orcid-web/src/main/java/org/orcid/frontend/web/controllers/WorksController.java

```diff
@@ -327,19 +327,6 @@ WorkForm getWorkInfo(@RequestParam(value = "workId") Long workId) {
                             ProfileEntity profileEntity = profileEntityCacheManager.retrieve(contributorOrcid);
                             String publicContributorCreditName = cacheManager.getPublicCreditName(profileEntity);
                             contributor.setCreditName(Text.valueOf(publicContributorCreditName));
-                            if(contributorOrcid.equals(getEffectiveUserOrcid())) {                                
```

Comment:
One solution possible solution....

1. Never return email address
2. If it has an valid ORCID id return the same name as you would source.
3. If there is no ORCID just return the name provided.

...just thinking